src/graphics.cpp

```diff
@@ -207,6 +441,45 @@ void GetvScreenAverage2()
 
     vScreenX[1] = (vScreenX[1] / B) + (ScreenW * 0.5);
     vScreenY[1] = (vScreenY[1] / B) + (ScreenH * 0.5) - vScreenYOffset;
+
+#ifdef __3DS__
+    vScreenX[1] -= std::fmod(vScreenX[1], 2.);
+    vScreenY[1] -= std::fmod(vScreenY[1], 2.);
+#endif
+}
+
+// NEW: Get the average screen position for all players with no level edge detection if it were 800x600, and write the top-left coordinate to (left, top)
+void GetvScreenAverage2Canonical(int* left, int* top)
+{
+    // int A = 0;
+    int B = 0;
+    double l = 0;
+    double t = 0;
+
+    for(int A = 1; A <= numPlayers; A++)
+    {
+        if(!Player[A].Dead)
+        {
+            l += -Player[A].Location.X - Player[A].Location.Width / 2.0;
+            if(Player[A].Mount == 2)
+                t += -Player[A].Location.Y;
+            else
+                t += -Player[A].Location.Y - Player[A].Location.Height;
+            B += 1;
```

Comment:
Maybe `++B`?

Reply:
I normally use postfix for primitives and prefix for literals, but I can use prefix here if you prefer. Agree that both are preferred to `+= 1`.